app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -121,16 +121,18 @@ class BrowserFragment : IWebViewLifecycleFragment() {
                 this@BrowserFragment.url?.let { url ->
                     when (value) {
                         NavigationEvent.VAL_CHECKED -> {
-                            CustomTilesManager.getInstance(context).pinSite(context, url,
-                                    webView?.takeScreenshot())
+                            browserOverlay.addPinnedSiteToTiles(CustomTilesManager.getInstance(context).pinSite(context, url,
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.